Jetsnack/app/src/main/java/com/example/jetsnack/ui/utils/SystemUi.kt

```diff
@@ -141,10 +219,10 @@ private class SystemUiControllerImpl(private val window: Window) : SystemUiContr
 }
 
 /**
- * An [androidx.compose.runtime.Ambient] holding the current [SysUiController]. Defaults to a
+ * An [androidx.compose.runtime.Ambient] holding the current [AmbientSysUiController]. Defaults to a
  * no-op controller; consumers should [provide][androidx.compose.runtime.Providers] a real one.
  */
-val SysUiController = staticAmbientOf<SystemUiController> {
+val AmbientSysUiController = staticAmbientOf<SystemUiController> {
```

Comment:
This should probably be be called `AmbientSystemUiController`

Reply:
Also, what do you think to the name `SystemUiController`? Is `SystemBarsController` a better name?

Reply:
Done